selftests/functional/test_basic.py

```diff
@@ -912,6 +913,69 @@ def test_disable_plugin(self):
                              (expected_rc, result))
             self.assertNotIn("Collect system information", result.stdout)
 
+    def test_plugin_order(self):
+        """
+        Tests plugin order by configuration file
+
+        First it checks if html, json, xunit and zip_archive plugins are enabled.
+        Then it runs a test with zip_archive running first, which means the html,
+        json and xunit output files do not make into the archive.
+
+        Then it runs with zip_archive set to run last, which means the html,
+        json and xunit output files *do* make into the archive.
+        """
+        os.chdir(basedir)
+        cmd_line = './scripts/avocado plugins'
+        result = process.run(cmd_line, ignore_status=True)
+        expected_rc = exit_codes.AVOCADO_ALL_OK
+        self.assertEqual(result.exit_status, expected_rc,
+                         "Avocado did not return rc %d:\n%s" %
+                         (expected_rc, result))
+        self.assertIn("html", result.stdout)
```

Comment:
Would you please remove the `html` plugin from this test? As an optional plugin it's not available during `make rpm`.